core/field_dropdown.js

```diff
@@ -217,6 +218,7 @@ Blockly.FieldDropdown.prototype.widgetCreate_ = function() {
     menuItem.setChecked(value == this.value_);
     if (value == this.value_) {
       selectedMenuItem = menuItem;
```

Comment:
No need to keep the local selectedMenuItem. You can replace it with this.selectedMenuItem_

Reply:
Made this change as well!